internal/controllers/suite_test.go

```diff
@@ -74,13 +74,17 @@ func newClient(t *testing.T) client.Client {
 
 type MockInstalledBundleGetter struct {
 	bundle *controllers.InstalledBundle
+	err    error
 }
 
 func (m *MockInstalledBundleGetter) SetBundle(bundle *controllers.InstalledBundle) {
 	m.bundle = bundle
 }
 
 func (m *MockInstalledBundleGetter) GetInstalledBundle(ctx context.Context, ext *ocv1.ClusterExtension) (*controllers.InstalledBundle, error) {
+	if m.err != nil {
+		return m.bundle, m.err
+	}
 	return m.bundle, nil
```

Comment:
```suggestion
	return m.bundle, m.err
```

@rashmi43 here @azych suggests that we can simplify and return the error directly. We do not need to check if is nil or not. if that is nill will return nill

Reply:
done